### ISSUE TITLE
Right sidebar - shortcuts don't scroll

### ISSUE THREAD

**Issue by TfTHacker** [mgsloan/roam-navigator#5]
Just a minor bug: if you scroll the left panel, the key shortcuts for favorite pages stay fix; then you can't tell which shortcut is for what page.

Mike, nice updates to the code. The Roam42 user bases is loving your work.

**Comment by mgsloan** [comment]
Yeah, I noticed that, but never figured out a straightforward way to fix it.

Workaround is to not have that many starred pages ;)

**Comment by TfTHacker** [comment]
This is a good solution :-)
